presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -154,6 +157,22 @@ public void testToUnixTime()
     }
 
     @Test
+    public void testFromISO8601()
+    {
+        assertFunction("from_iso8601_timestamp('" + TIMESTAMP.toString(ISODateTimeFormat.dateTime()) + "')", TIMESTAMP_WITH_TIME_ZONE, toTimestampWithTimeZone(TIMESTAMP_WITH_NUMERICAL_ZONE));
```

Comment:
The argument to the function should be a constant. Relying on the output of TIMESTAMP.toString() might mask issues if that call is sensitive to the VM timezone, etc.